rpcs3/Emu/Cell/SPURecompiler.cpp

```diff
@@ -7749,9 +7749,9 @@ class spu_llvm_recompiler : public spu_recompiler_base, public cpu_translator
 
 			const auto ma = eval(sext<s32[4]>(fcmp_uno(a != fsplat<f32[4]>(0.))));
 			const auto mb = eval(sext<s32[4]>(fcmp_uno(b != fsplat<f32[4]>(0.))));
-			const auto ca = eval(bitcast<f32[4]>(bitcast<s32[4]>(a) & mb));
-			const auto cb = eval(bitcast<f32[4]>(bitcast<s32[4]>(b) & ma));
-			set_vr(op.rt, fm(ca, cb));
+			const auto cx = eval(ma & mb);
```

Comment:
I wonder if it could be more correct to `&` first, then use `sext`, but I won't bother trying it for now.